tests/core/go_test/example_test.go

```diff
@@ -0,0 +1,26 @@
+package lib
+
+import "fmt"
+
+var Expected = "Expected"
+
+func ExampleHelloWorld() {
+	fmt.Println("Hello Example!")
+	fmt.Println("expected: " + Expected)
+	fmt.Println("got: " + Got)
+	// Output:
+	// Hello Example!
+	// expected: Example
+	// got: Example
+}
+
+func ExampleDontTestMe() {
+	fmt.Println("Dont Test Me!")
```

Comment:
Maybe replace this with a `panic`. I'm sure that would be reported if it were run.